internal/runbits/refresh.go

```diff
@@ -15,7 +16,11 @@ import (
 
 // RefreshRuntime should be called after runtime mutations.
 func RefreshRuntime(auth *authentication.Auth, out output.Outputer, an analytics.Dispatcher, proj *project.Project, commitID strfmt.UUID, changed bool, trigger target.Trigger, svcm *model.SvcModel) (rerr error) {
-	target := target.NewProjectTarget(proj, &commitID, trigger)
+	err := buildscript.Sync(proj, &commitID, out, auth)
```

Comment:
Thinking on this some more I would expect to see this function call, or something similar, more often. For example: If the user modifies their local order file and then runs `state activate` I would expect the runtime to be updated. However, I don't believe `RefreshRuntime` will be called in that case, so the changes to the local order file will not be picked up. I could be missing something but I just want to call it out so we can ensure we've got that covered.

Reply:
`state activate` calls `runtime.NewFromProject()`, which syncs the build script. I did a search for all instances we call `runtime.New()` and this PR calls `buildscript.Sync()` for all of them except `state _prepare` and `state deploy`. I'm not sure if it's necessary for the former, and deployments are not projects, so syncing doesn't seem necessary there.

Does this assuage your concerns? Or am I not looking in the right place?

Reply:
Great! I just wanted to make sure that we've covered all the uses of the runtime and you have done that.